.github/workflows/type_checks.yml

```
@@ -50,7 +50,7 @@ jobs:
           # 4. other packages that are used somewhere in PyGMT
           python -m pip install \
             numpy pandas xarray netcdf4 packaging \
-            contextily geopandas ipython rioxarray \
+            contextily geopandas ipython pyarrow rioxarray \
             mypy pandas-stubs \
```

Add [pyarrow-stubs](https://github.com/zen-xu/pyarrow-stubs) as well?
```suggestion
            mypy pandas-stubs pyarrow-stubs \
```